packages/plugin-rok/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@jspsych-contrib/plugin-rok",
+  "version": "1.0.0",
```

Comment:
Drop this to version 0.1.0 so that the release process can automatically bump it to 1.0